bin/interactive.py

```diff
@@ -189,52 +177,59 @@ def mkpipe():
     sys.stderr.flush()
     new = l=='\n'
 '''
+
+    validator = subprocess.Popen(
+        validator_command,
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        # TODO: Make a flag to pass validator error directly to terminal.
+        stderr=subprocess.PIPE if validator_error is False else None,
+        bufsize=BUFFER_SIZE,
```

Comment:
I'm pretty sure we want `pipesize` here and elsewhere. `bufsize` is the python-internal read/write buffer size, not the size of the pipe. Not sure what kind of buffering we should want; i'm inclined to say none

Reply:
We can't use `pipesize` yet, because that's only introduced in Python 3.10 (that's why I added the `set_pipe_size` function to set the pipe size manually). I'll test it out without the `bufsize` parameter, perhaps it is not needed indeed

Reply:
oh right yes :P

Reply:
It appears to work! :smile: (tested on the same problems as https://github.com/RagnarGrootKoerkamp/BAPCtools/pull/251#issuecomment-1609289822)